pages/learn/deploy/release-strategy/identical-releases-across-fleets.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Identical releases across multiple fleets
+excerpt: Use the same release across multiple fleets using release bundles
+---
+
+# Identical releases across multiple fleets
```

Comment:
To bring it as more of an action word, how about we do the title as 

Share releases with multiple fleets